osi_analytic_segments_purchase/README.rst

```diff
@@ -0,0 +1,26 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+
+==============================
+OSI Analytic Segments Purchase
+==============================
+
+This module adds 2 additional analytic segments to Purchase related models and views
+
+Configuration
+=============
+
+* Create Segments at Accounting/Configurations/Segments
+
+Credits
+=======
+
+* Mayank Gosai <mgosai@ursainfosystems.com>
+* Bhavesh Odedra <bodedra@opensourceintegrators.com>
+
+Contributors
+------------
+
+* Open Source Integrators <http://www.opensourceintegrators.com>
```

Comment:
Same here, Swap the names between contributors and credits.

Reply:
Done